FlashEnglish/Sources/Views/Score/ScoreView.swift

```diff
@@ -29,6 +34,11 @@ struct ScoreView: View {
         .navigationBarBackButtonHidden()
     }
 
+
+
+
+    
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/f0c1bb17c92f6625d748b7be15707d1134141005/FlashEnglish/Sources/Views/Score/ScoreView.swift#L41"></span>Limit vertical whitespace to a single empty line; currently 6<br />
<code>vertical_whitespace</code> <code>ScoreView.swift:41</code></td>
    </tr>
  </tbody>
</table>